python/lsst/pipe/tasks/skyCorrection.py

```diff
@@ -366,6 +367,7 @@ def run(self, calExps, calBkgs, skyFrames, camera, backgroundToPhotometricRatioH
             _ = self._restoreOriginalBackgroundRefineMask(calExp, calBkg)
             masks.append(calExp.mask)
             skyCorrs.append(calBkg)  # Contains only the inverted original background elements at this stage
+            bgModel1Indices.append(len(calBkg._backgrounds))  # Index of the original background element
```

Comment:
Anyway you can do this without using the attribute with a leading underscore?

Reply:
It looks like `len(calBkg)` should just work.

Reply:
Yes, I think you're right, happy to switch to just `len(calBkg)`